quarkus-opentelemetry-exporter-common/src/main/java/io/quarkiverse/opentelemetry/exporter/common/runtime/LateBoundSpanProcessor.java

```diff
@@ -17,6 +17,14 @@ public class LateBoundSpanProcessor implements SpanProcessor {
     private boolean warningLogged = false;
     private SpanProcessor delegate;
 
+    public LateBoundSpanProcessor() {
```

Comment:
Can the delegate be null?

Reply:
Nice catch... Will remove because it's not used.